app/_src/policies/meshtimeout.md

```diff
@@ -280,8 +291,55 @@ spec:
 {% endpolicy_yaml %}
 {% endif_version %}
 
+{% if_version gte:2.6.x %}
+#### Default configuration for all gateways in the Mesh
+
+This configuration will be applied on inbounds and outbounds of all gateways.
+
+{% policy_yaml example7 %}
+```yaml
+type: MeshTimeout
+name: mesh-gateways-timeout-all-default
+mesh: default
+spec:
+  targetRef:
+    kind: Mesh
+    proxyTypes: ["Gateway"]
+  from:
+    - targetRef:
+        kind: Mesh
+      default:
+        idleTimeout: 5m
+        http:
+          streamIdleTimeout: 5s
+          requestHeadersTimeout: 500ms
+  to:
+    - targetRef:
+        kind: Mesh
+      default:
+        idleTimeout: 1h
+        http:
+          streamIdleTimeout: 5s
+```
+{% endpolicy_yaml %}
+{% endif_version %}
+
 ### Defaults
 
+{% if_version lte:2.5.x %}
+
+| Property                   | default |
+|----------------------------|---------|
+| idleTimeout                | 1h      |
+| connectionTimeout          | 5s      |
+| http.requestTimeout        | 15s     |
+| http.streamIdleTimeout     | 30m     |
+| http.maxStreamDuration     | 0s      |
+| http.maxConnectionDuration | 0s      |
```

Comment:
for tables you can use `inline: true` parameter just like I did here: https://github.com/kumahq/kuma-website/pull/1578/files#diff-5d39abc995a519665f471d592cfeb35220a2df1e4b76a648a479e63281685888R19 to not have to repeat the whole table